stories/buttons.stories.tsx

```diff
@@ -65,15 +53,17 @@ const DefaultStory: React.FC = () => {
             <Button disabled variant="secondary">
               Secondary
             </Button>
+            <Button variant="secondary" color="text">
```

Comment:
I believe the color type should be specified somewhere in the storybook, otherwise I don't know what I'm looking at  if I don't see the code(I see 2 disabled buttons with exact same styling).

Reply:
That's a good idea